gtsfm/utils/pycolmap_utils.py

```diff
@@ -97,7 +97,7 @@ def gtsfm_calibration_to_colmap_camera(
     """
     if isinstance(calibration, gtsam.Cal3Bundler):
         return ColmapCamera(
-            model="SIMPLE_RADIAL",
+            model="RADIAL",
```

Comment:
@travisdriver @akshay-krishnan I changed this, as Cal3Bundler has a k2. Or am I missing something?